src/actions/runJavascriptOnWebPage.ts

```diff
@@ -0,0 +1,44 @@
+import { withActionOutput } from '../utils';
+
+import WFSerialization from '../interfaces/WF/WFSerialization';
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Match Text Action. Searches text passed into the action for matches to a regular expression.
+ *
+ * ```js
+ * runJavascriptOnWebPage({
+ *   text: 'var result = [];
```

Comment:
The `text` parameter should be a templateString if it has line breaks.
Could you change it to the following, please?

```ts
/**
 * Match Text Action. Searches text passed into the action for matches to a regular expression.
 *
 * ```js
 * runJavascriptOnWebPage({
 *   text: `var result = [];
 *   // Get all links from the page
 *   var elements = document.querySelectorAll('a');
 *   for (let element of elements) {
 *     result.push({
 *       url: element.href,
 *       text: element.innerText
 *     });
 *   }
 *
 *   // Call completion to finish
 *   completion(result);`
 * });
 * ```
 */
```